### PR TITLE
Prepare 2.3.0

### PR DESCRIPTION
**Pull request by fpetkovski**
**What this PR does / why we need it**:
Prepares the release of KSM 2.3.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## v2.3.0 / 2021-12-09
+
+[FEATURE]     Add a `--namespace-denylist` command line flag to exclude metrics from certain namespaces #1596 #1595 @mallow111
+[FEATURE]     Add `kube_*_labels` and `kube_*_annotations` metrics for Pod Disruption Budgets #1623 @arajkumar
+[FEATURE]     Add a Kustomization file for deploying KSM in authosharding mode #1603 @karancode
+[FEATURE]     Expose a metric for the number of ports in endpoint objects #1571 @bavarianbidi
+[FEATURE]     Add a command line flag for opt-in metrics #1643 @Serializator
+[FEATURE]     Add `kube_horizontalpodautoscaler_info` metric
```
Comment (mrueg) [inline CHANGELOG.md:8]: Can you add  #1648 here?

**Review by mrueg — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (fpetkovski) [reply CHANGELOG.md]: Thanks, added

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## v2.3.0 / 2021-12-09
+
+[FEATURE]     Add a `--namespace-denylist` command line flag to exclude metrics from certain namespaces #1596 #1595 @mallow111
```
Comment (dgrisonnet) [inline CHANGELOG.md:3]: The formatting is broken, could we perhaps add bullets points in a similar way as what we did for the previous versions?

Comment (fpetkovski) [reply CHANGELOG.md]: Sure, I completely missed the bulleted points. Added

**Pull request merged**